iothub/device/tests/Transport/Mqtt/MqttTransportHandlerTests.cs

```diff
@@ -357,19 +359,21 @@ public async Task MqttTransportHandlerEnableTwinPatchAsyncTimesOut()
         // Tests_SRS_CSHARP_MQTT_TRANSPORT_18_034: `SendTwinGetAsync` shall shall open the transport  if this method is called when the transport is not open.
         // Tests_SRS_CSHARP_MQTT_TRANSPORT_18_021: If the response contains a success code, `SendTwinGetAsync` shall return success to the caller
         [TestMethod]
-        public async Task MqttTransportHandlerSendTwinGetAsyncHappyPath()
+        [DataRow("foo", TwinJsonString1)]
+        [DataRow("Iso8601String", TwinJsonString2)]
+        public async Task MqttTransportHandlerSendTwinGetAsyncHappyPath(string propName, string twinJson)
```

Comment:
By calling `SendTwinGetAsync()` on L383, it can validate `JsonConvert.DeserializeObject()` working properly with date-formatted string containing trailing zeros.